packages/cardano-services-client/src/AssetInfoProvider/assetInfoHttpProvider.ts

```diff
@@ -10,6 +15,24 @@ const paths: HttpProviderConfigPaths<AssetProvider> = {
   healthCheck: '/health'
 };
 
+const isAssetInfo = (assetInfo: Asset.AssetInfo | unknown): assetInfo is Asset.AssetInfo =>
```

Comment:
That's probably equivalent
```suggestion
const isAssetInfo = (assetInfo: unknown): assetInfo is Asset.AssetInfo =>
```

Reply:
Done in [6e28df4](https://github.com/input-output-hk/cardano-js-sdk/pull/696/commits/6e28df412797974b8ce6f6deb0c3346ff5938a05)